.travis.yml

```diff
@@ -53,7 +53,7 @@ deploy:
     repo: kubevirt/containerized-data-importer
  # Release versioned images
 - provider: script
-  script: env CSV_VERSION=${TRAVIS_TAG#v} make publish
+  script: env CSV_VERSION=${TRAVIS_TAG#v} make olm-push publish
```

Comment:
Where are `CSV_VERSION`, `QUAY_USERNAME`, ... set?

Reply:
CSV_VERSION is set right there, env CSV_VERSION=${TRAVIS_TAG#v} (#v strips the V), The username/pw are set in the travis configuration. When the build is triggered they are inserted into the environment (also since they are marked secret cannot be printed in the build).